src/VisualStudio/Core/Impl/CodeModel/FileCodeModel.cs

```diff
@@ -329,8 +330,13 @@ internal void PerformEdit(Func<Document, Document> action)
 
                 var result = action(document);
 
-                var formatted = Formatter.FormatAsync(result, Formatter.Annotation).WaitAndGetResult_CodeModel(CancellationToken.None);
-                formatted = Formatter.FormatAsync(formatted, SyntaxAnnotation.ElasticAnnotation).WaitAndGetResult_CodeModel(CancellationToken.None);
+                var formatted = State.ThreadingContext.JoinableTaskFactory.Run(async () => 
+                {
+                    var formatted = await Formatter.FormatAsync(result, Formatter.Annotation).ConfigureAwait(false);
+                    formatted = await Formatter.FormatAsync(formatted, SyntaxAnnotation.ElasticAnnotation).ConfigureAwait(false);
```

Comment:
```suggestion
                    var formatted = await Formatter.FormatAsync(result, Formatter.Annotation).ConfigureAwait(true);
                    formatted = await Formatter.FormatAsync(formatted, SyntaxAnnotation.ElasticAnnotation).ConfigureAwait(true);
```

There is no advantage to using `ConfigureAwait` inside a `Run` invocation (the current thread will be blocked by `Run`, so it's not only fine to allow continuations to execute on it, but can actually perform better since there is no waiting. Since there is an analyzer false-positive for this case, it can be suppressed by explicitly using `ConfigureAwait(true)`.